my-app/app/components/create-grp.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    users: [],
+    actions: {
+        addUser: function () {
+            var user = {};
+            user.name = this.get('name');
+            user.phone = this.get('phone');
+            user.hasReceived = "";
+            user.hasPaid = "";
+            this.get('users').pushObject(user);
```

Comment:
The validation is happening on server instead of that you can check here itself if the new user exists already.